cfgmgr/buffer_pool_mellanox.lua

```diff
@@ -158,7 +158,18 @@ local function fetch_buffer_pool_size_from_appldb()
         end
         xoff = redis.call('HGET', 'BUFFER_POOL_TABLE:' .. buffer_pools[i], 'xoff')
         if not xoff then
-            table.insert(result, buffer_pools[i] .. ':' .. size)
+            if shp_enabled and size == "0" and buffer_pools[i] == "ingress_lossless_pool" then
```

Comment:
can a mock test be added for this?

Reply:
Hi @neethajohn 
We can't mock test for a Lua script since it will not be invoked in a mock test context.
I added a vs test. Please check it.
Thanks